src/classes/whexcel.py

```diff
@@ -132,22 +133,24 @@ def export_sheet(self, book, sheet_name, widths, a_df):
                 sheet.set_column(idx, idx, width)
 
         #title row
-        sheet.write_row(0, 0, [c.replace('_',' ').title() for c in a_df.columns])
+        sheet.write_row(0, 0, [c.replace('_', ' ').title() for c in a_df.columns])
 
         #data rows
         for idx, row_data in enumerate(a_df.itertuples(index=False)):
             sheet.write_row(idx + 1, 0, row_data, None)
 
-    def write_hyperlink(self, worksheet, row, col, link:Hyperlink, _):
-        '''write_xlsx_hyperlink'''
+    @staticmethod
+    def write_hyperlink(worksheet, row, col, link: Hyperlink, _):
+        """write_xlsx_hyperlink"""
         return worksheet.write_url(row, col, url=link.url, string=link.title)
 
-    def write_local_datetime(self, worksheet, row, col, ts, _):
-        '''
+    @staticmethod
```

Comment:
there are a couple methods in here where `self` was left unreferenced, so they can be made to be static methods.